specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/examples/communityGalleryExamples/CommunityGallery_Get.json

```diff
@@ -13,6 +13,21 @@
         "type": "Microsoft.Compute/Locations/CommunityGallery",
         "identifier": {
           "uniqueId": "/CommunityGalleries/publicGalleryName"
+        },
+        "properties": {
+          "artifactTags": {
+            "ShareTag-CommunityGallery": "CommunityGallery"
+          },
+          "disclaimer": "https://aka.ms/community-gallery-disclaimer",
+          "communityMetadata": {
+            "publisherUri": "http://microsoft.com",
+            "publisherContact": "sig@microsoft.com",
```

Comment:
This email address is real, but currently invalid, and may encourage needless spam to MS servers... or become a valid email in the future?? Consider using an actual fake address.